internal/handler/wallet/api_test.go

```diff
@@ -69,14 +70,17 @@ func MockAPI() (*operations.MassaWalletAPI, chan PasswordPrompt, chan PrivateKey
 	// Create a new MassaWalletAPI instance
 	massaWalletAPI := operations.NewMassaWalletAPI(swaggerSpec)
 
-	mockChanPassword := make(chan PasswordPrompt, 2) // buffered channel
+	mockChanPassword := make(chan PasswordPrompt, 3) // buffered channel
```

Comment:
do we have any idea how to define the optimal buffer size ? why 3 ?

Reply:
no idea sorry, I put 3 because 2 didn't work and I didn't dig that much into this due to lack of time